modules/webrtcvad/vad.py

```
@@ -0,0 +1,80 @@
+        if len(audio.main) < frame_size_bytes:
+            raise ValueError(
+                f"Audio data is too short. Expected at least {frame_size_bytes} bytes, but got {len(audio.main)} bytes."
+            )
+
+        buffer.seek(-frame_size_bytes, io.SEEK_END)
+        audio_data = buffer.read(frame_size_bytes)
+
+        try:
+            is_speech = self._vad.is_speech(audio_data, params.sample_rate)
+        except Exception as e:
+            raise ValueError(f"Error processing audio data with WebRTC VAD: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)